src/tool.js

```diff
@@ -0,0 +1,30 @@
+exports.buildForm = tools => {
+  let copy = "";
+  try {
+    tools.forEach(tool => {
+      if (tool.type) {
+        switch (tool.type) {
```

Comment:
At the end of your switch statement, make sure to create a "default" for the switch return.